examples/FailingTests.elm

```diff
@@ -86,4 +87,31 @@ testView =
                         |> Query.find [ tag "ul" ]
                         |> Query.findAll [ tag "li" ]
                         |> Query.each (Query.has [ classes [ "list-item", "themed" ] ])
+            , test "expect first a to send SomeMsg onClick" <|
+                \() ->
+                    output
+                        |> Query.findAll [ tag "a" ]
+                        |> Query.first
+                        |> Events.msgFor "click" "{}"
+                        |> Expect.equal (Ok SomeMsg)
```

Comment:
I know this is not the usual API for this lib, usually the function already receives an expectation, like `Query.count (Expect.equal 5)`, but I was thinking of a later usage, where I want to get the event from the element, but I don't actually want to write my assertion with it, just update the state with the returned Msg and maybe even trigger another event in sequence.

Returning like this allows me to do both, Expect as shown above and use the value for other things.